### PR TITLE
fix(#1112): generate code for parameter values with spaces

### PR DESCRIPTION
**Pull request by afonsocarlos**
# Description

This fixes the error "Error generating code snippet" when Environment parameters have values with spaces

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [ ] **Create an issue and link to the pull request.**

### CODE REVIEW

File: packages/bruno-app/src/utils/codegenerator/har.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const createPostData = (body) => {
       mimeType: contentType,
       params: body[body.mode]
         .filter((param) => param.enabled)
-        .map((param) => ({ name: param.name, value: param.value }))
+        .map((param) => ({ name: param.name, value: encodeURIComponent(param.value) }))
```
Comment (martinsefcik) [inline packages/bruno-app/src/utils/codegenerator/har.js]: Theoretically param name can also contains spaces or some other characters which should be encoded (at least I'm not aware of such RFC which does not allow it). So don't we want to apply the same also for param name ?

Comment (afonsocarlos) [reply packages/bruno-app/src/utils/codegenerator/har.js]: We certainly could. I've never come across such case, but escaping the name should do no harm anyways.

Comment (afonsocarlos) [reply packages/bruno-app/src/utils/codegenerator/har.js]: @martinsefcik on second thought, it makes more sense to just encode the whole URL. That would catch any corner cases. Does that make sense for you?

**Comment by sanjai0py** [comment]
Closing this PR as another [PR](https://github.com/usebruno/bruno/pull/1827) with the same changes has already been merged. Thanks for working on this.

**Pull request closed without merging**